### PR TITLE
Feature: Add podyaml and nodeyaml panel in dashboard and corresponding API

### PR DESCRIPTION
**Pull request by wuchang0201**
Feature show podyaml and nodeyaml, custom log

### CODE REVIEW

**Comment by wuchang0201** [comment]
example：
image

File: build/docker/Dockerfile.grafana [hunk 1]
```diff
@@ -2,28 +2,22 @@
 FROM node as node
 # 将代码拷贝到镜像中的工作目录
 COPY ./grafana_plugin/yaml_panel /app/yaml_panel
-COPY ./grafana_plugin/custom_log_panel /app/custom_log_panel
```
Comment (d3c3mber) [inline build/docker/Dockerfile.grafana:5]: why remove this plugin?

**Comment by d3c3mber** [comment]
can you please squash the commits into one? thanks

**Review by d3c3mber — APPROVED** [review]

**Review by linuzb — APPROVED** [review]
LGTM

**Comment by wuchang0201** [comment]
#81

**Pull request merged**